src/crate/templates/statefulset.yaml

```diff
@@ -46,6 +49,8 @@ spec:
             {{- if .Values.resources.limits.cpu }}
             - -Cprocessors="$MIN_VCPU_COUNT"
             {{- end }}
+          securityContext:
+            privileged: true
```

Comment:
We should use the same guard as above here, to avoid it in environments that don't need it.

That said... [this chroot](https://github.com/crate/docker-crate/blob/master/docker-entrypoint.sh#L21)? Seems like we should just `runAsUser: 1000` to achieve the same but avoid that code path?